lib/prompt.js

```diff
@@ -1,73 +1,61 @@
-/* eslint-disable no-console */
 const inquirer = require('inquirer');
 const fixture = require('./fixture.js');
 const helpers = require('./helpers.js');
-require('./theme.js');
+const { print } = require('./theme.js');
 
-function retrieveFixture(program, path, cb) {
+async function retrieveFixture(fixtureKey, path) {
   const componentPath = helpers.getComponentPath(path);
   const fixtures = fixture.readFixture(componentPath);
 
-  if (!fixtures) cb();
-
-  const fixtureKey = program.fixture;
-
-  const findFixtureByKey = (key) => {
-    const result = fixtures[key];
-    if (!result) {
-      console.log("Fixture '%s' does not exist".error, key);
-      helpers.destroyProcess();
-      cb();
-    }
-    cb(result);
-  };
+  if (!fixtures) {
+    print('No fixtures found', ERROR);
+    return;
```

Comment:
perhaps better to throw an error since `retrieveFixture` is awaited and expects a response